lib/reverse_words.rb

```diff
@@ -1,6 +1,34 @@
 # A method to reverse each word in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n), where n is the length of my_words
+# Space complexity: O(1)
 def reverse_words(my_words)
-  raise NotImplementedError
+    if my_words.nil?
+      return nil
+    end
+    i = 0
```

Comment:
Tiny nit (though it's ultimately a matter of personal/team preference): these variable names could be more descriptive. 🙂